game/models.py

```
@@ -136,4 +162,30 @@ class Meta:
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="featured_badges"
+    )
+
+    achievement = models.ForeignKey(
+        Achievement,
+        on_delete=models.CASCADE
+    )
+
+    def save(self, *args, **kwargs):
```

Ruff (0.15.15)

[warning] Missing type annotation for `*args`

(ANN002)